src/components/BooksList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
If you read the requirement again, you'll notice that these components
- `BookLists`
- `BookForm`
being "container" components should be in the `src/containers/` folder

This is a minor issue, but please relocate these two files for final thumbs up